src/commands/data/connectors/update.ts

```diff
@@ -9,10 +9,10 @@ interface Dict<T> {
 }
 
 export default class ConnectorsUpdate extends BaseCommand {
-  static description = 'Update the settings for a Data Connector\nRead more about this feature at https://devcenter.heroku.com/articles/heroku-data-connectors#update-configuration'
+  static description = 'Update the settings, tables and columns to exclude for a Data Connector\nRead more about this feature at https://devcenter.heroku.com/articles/heroku-data-connectors#update-configuration'
 
   static examples = [
-    '$ heroku data:connectors:update gentle-connector-1234 --setting key=value --setting otherKey=otherValue',
+    '$ heroku data:connectors:update gentle-connector-1234 --setting key=value --setting otherKey=otherValue --add-table public.posts --add-table public.people --remove-table public.people --add-exclude public.posts.key --remove-exclude public.posts.key',
```

Comment:
It's a bit confusing to add and remove the same objects in the same example command. We can include multiple examples. What do you think about separating them?

```suggestion
    '$ heroku data:connectors:update gentle-connector-1234 --setting key=value --setting otherKey=otherValue',
    '$ heroku data:connectors:update gentle-connector-1234 --add-table public.posts --add-table public.people',
    '$ heroku data:connectors:update gentle-connector-1234 --remove-table public.people',
    '$ heroku data:connectors:update gentle-connector-1234 --add-exclude public.posts.key',
    '$ heroku data:connectors:update gentle-connector-1234 --remove-exclude public.posts.key',
    '$ heroku data:connectors:update gentle-connector-1234 --setting key=value --add-table public.posts --add-exclude public.posts.key',
```

Reply:
Yea that is confusing. Fixing.